bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "perfkit-explorer-deps",
+  "version": "23.0.0",
+  "description": "Bower dev/test dependencies for PerfKit Explorer",
+  "author": "Joe Allan Muharsky <jmuharsky@gmail.com>",
+  "dependencies": {},
+  "devDependencies": {
+    "angular": "~1.4.1",
+    "angular-mocks": "~1.4.1",
+    "angular-material": "~0.10.0",
+    "angular-ui-router": "~0.2.15",
+    "angular-ui-grid": "~3.0.0-rc.22",
+    "angular-bootstrap": "~0.10.0",
+    "bootstrap-css-only": "~3.2.0",
+    "codemirror": "~5.3.0",
+    "jquery": "~2.1.4"
```

Comment:
No more dependencies are loaded from CDN's.  All scripts are now served locally due to flakiness/update issues with CDN particularly on ui-grid and angular-material.